### ISSUE TITLE
Analytics Implementation or QA Support for [Search & Discovery - Typeahead 2.0]

### ISSUE THREAD

**Issue by LindaLi123** [department-of-veterans-affairs/va.gov-team#31870]
# Google Analytics Implementation or QA Request

## What this form is for

<details>
<summary>Request Google Analytics Implementation</summary>
<div>
    <br>
<p>Use this template to request Google Analytics implementation or QA for your product. Please fill out all brackets.</p>
    
> _Please see the following information for implementation timeline:_
> New Teams: Implementation will usually be completed in 4 sprints
> 
> - Sprint 1: Discovery/Orientation
> - Sprint 2: FE Implementation
> - Sprint 3: Tagging Implementation
> - Sprint 4: QA
> 
> All other teams: Implementation will usually be completed in 2 sprints
> 
> - Sprint 1: Discovery and FE Implementation
> - Sprint 2: Tagging Implementation and QA

</div>
</details>

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [x] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**

    - [Search & Discovery Team]
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - [Staging Link](https://staging.va.gov/)
    
- **When is the launch date?**
    - [Insert launch date]

- **What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_
    
    - [If this is a funnel, insert each step's URL - pages visible depending on the applicant's answers should be marked accordingly]
    
    - [Insert other interactions - please review the [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) and [GA Data Dictionary](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Google-Analytics-Data-Dictionary.1810464912.html)  to make sure these interactions aren't already tracked]

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - [Insert test user]

- **Any additional comments**

    - [Insert comments]

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by LindaLi123** [comment]
@laurensimpson14 hello Lauren, would you pls advise your availability for a call?

**Comment by laurensimpson14** [comment]
@LindaLi123 Are we still working off of this ticket? I want to make sure I haven't missed any notifications. 

@srsuddath Let me know when the `location` parameter has been added 👍

**Comment by LindaLi123** [comment]
@laurensimpson14 Hello Lauren, yes we are still working off this ticket.

**Comment by LindaLi123** [comment]
@laurensimpson14 Hello Lauren Sam confirmed PR has been created, pls let me know if you need anything else from us?

**Comment by laurensimpson14** [comment]
@LindaLi123 I was waiting on that PR to get merged. I just checked, and it looks like it was merged this morning. I'll add the required tagging updates now and update you when complete.

**Comment by laurensimpson14** [comment]
@bsmartin-ep I added a `search-location` CD to the `EV - Search Performed - From Search Dropdown - ONCE PER PAGE` event. Could you please QA? Found in `31870 - Search Location CD` [workspace in GTM](https://tagmanager.google.com/?utm_source=marketingplatform.google.com&utm_medium=et&utm_campaign=marketingplatform.google.com%2Fabout%2Ftag-manager%2F#/container/accounts/144676348/containers/2214120/workspaces/1000570). (For reference [here is the PR](https://github.com/department-of-veterans-affairs/vets-website/pull/19363) created by Sam.)

**Comment by bsmartin-ep** [comment]
This LGTM @laurensimpson14.  Should I publish?

**Comment by laurensimpson14** [comment]
@bsmartin-ep Yes please! 🚀

**Comment by bsmartin-ep** [comment]
We are live.  OK to close this issue?

**Comment by LindaLi123** [comment]
Yes Thank you

**Comment by laurensimpson14** [comment]
@LindaLi123 [Here is a report ](https://analytics.google.com/analytics/web/#/report/content-event-events/a50123418w177519031p176188361/_u.date00=20211119&_u.date01=20211119&explorer-table.plotKeys=%5B%5D&explorer-table.secSegmentId=analytics.customDimension93&_r.drilldown=analytics.eventCategory:Search) showing search results with the new `Search Location` custom dimension added to the report. This new custom dimension will be available point forward and not retroactively. So if you change the date range to prior to 11/19 then no data will show. 

Let me know if you have any questions!
